rgw/v2/tests/s3_swift/test_compression_encryption.py

```diff
@@ -0,0 +1,209 @@
+"""
+test_compression_encryption.py - using boto
+
+Usage: test_compression_encryption.py -c <input_yaml>
+<input_yaml>
+    test_compression_encryption.yaml
+    test_s3_copy_compression_encryption.yaml
```

Comment:
@viduship , could you update the details with the conf files

Reply:
updated @mkasturi18